Marlin/language_fr.h

```diff
@@ -189,7 +192,6 @@
 #if LCD_WIDTH > 19
   #define MSG_INFO_PRINT_COUNT              "Nbre impressions"
   #define MSG_INFO_COMPLETED_PRINTS         "Terminees"
-  #define MSG_INFO_PRINT_TIME               "Tps impr. total"
```

Comment:
`MSG_INFO_PRINT_TIME` is [still alive in `language_en.h`](https://github.com/MarlinFirmware/Marlin/blob/RCBugFix/Marlin/language_en.h#L498-L500).
[Removed string is `MSG_PRINT_TIME`](https://github.com/MarlinFirmware/Marlin/commit/4be635ec6e7499c89db5d36b7f2421421fef3792#diff-60ae373faf7c0ef13394687a3e476bddL503).